### PR TITLE
Journalist can add images to article 

### PR DESCRIPTION
**Pull request by shahin1290**
## Pivotal tracker
[Journalist can add images to article ](https://www.pivotaltracker.com/story/show/163088252)

## Tasks
- Adds unit test 
- Install active storage
- Updates model to attach image
- Adding images in the form
- Store the image to AWS cloud

## Gem
- aws-sdk-s3

## ScreenShots
screenshot from 2019-01-17 20-23-40
screenshot from 2019-01-17 20-23-54

### CODE REVIEW

**Review by johannaanden — APPROVED** [review]
LGTM

**Review by CamronLDNF — COMMENTED** [review]
Looks good. No comments from me.

File: features/Journalist_can_create_article.feature [hunk 1]
```diff
@@ -21,6 +21,7 @@ Feature: Journalist can create article
         Given I fill in 'Title' field with 'My article'
         And I fill in 'Description' field with 'This is a description'
         And I fill in 'Content' field with 'This is my article about the sunny weather'
+        And I attach a file
```
Comment (oliverochman) [inline features/Journalist_can_create_article.feature:24]: Add this step to the `Journalist_can_update_article` feature test as well

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: features/Journalist_can_create_article.feature (hunk 1 above)
Comment (shahin1290) [reply features/Journalist_can_create_article.feature:24]: It's been updated

**Review by oliverochman — APPROVED** [review]
LGTM!

**Pull request merged**